app/src/main/java/com/sq/thed_ck_licker/ecs/systems/cardSystems/CardCreationSystem.kt

```diff
@@ -241,12 +242,30 @@ class CardCreationSystem @Inject constructor(
                     name = "Time Bound Card", hp = 183f, score = 10000
                 )
             )(cardId)
-            cardId add TickComponent(1000)
+            cardId add TickComponent(
+                tickThreshold = 1000
+            )
             cardId add TriggeredEffectsComponent(
                 mutableMapOf(
                     Trigger.OnPlay to mutableListOf(
```

Comment:
This is the comparison part.
The old side is consice, but the new style is quite atomic.
Maybe there is some dsl syntax that we will figure out in future, 
Mean while i think it is acceptable and describes the effects well.
Speaking of descriptions, they are not up to date, as they still need time in the think tank.